src/utils/load-xcode.js

```diff
@@ -9,12 +9,15 @@ const { color, fatal } = require('./logging');
 const Xcode = require('./xcode');
 const evmConfig = require('../evm-config');
 
-function loadXcode(quiet = false) {
+function loadXcode(options) {
```

Comment:
```suggestion
function loadXcode(options = {}) {
```

To handle cases where no options are provided, like here:

https://github.com/electron/build-tools/blob/28bf599df3ca3c38ba68c79978f8b73ff0ae9eed/src/e#L187